segmentation/utils/visualize_test.py

```diff
@@ -10,3 +11,17 @@ def test_visualize_z_scores():
     # Assign random phenotype titles
     pheno_titles = [chr(i) for i in range(ord('a'), ord('z') + 1)]
     plot = visualize.visualize_z_scores(z, pheno_titles)
+    
+def test_visualize_cells():
+    rand_type = []
+    ids = []
+    for x in range(0,1000):
+        rand_type.append(chr(random.randint(0,25) + 97))
+        ids.append(random.randint(1,5))
```

Comment:
typically, arrays/lists in python aren't initialized this way.  some one-liners for these could look like:
```
rand_type = random.choices(string.ascii_lowercase, k=1000)
ids = random.choices(range(1,6), k=1000)
```
(might have to `import string.ascii_lowercase` )